third_party/move/move-compiler-v2/src/env_pipeline/inliner.rs

```diff
@@ -710,11 +820,51 @@ impl<'env, 'rewriter> InlinedRewriter<'env, 'rewriter> {
 
         let lambda_param_map: BTreeMap<Symbol, &Exp> = lambda_args_matched
             .iter()
-            .map(|(param, arg_exp)| (param.0, *arg_exp))
+            .map(|(param, arg_exp)| (param.1 .0, *arg_exp))
             .collect();
 
-        let (regular_params, regular_actuals): (Vec<&Parameter>, Vec<&Exp>) =
+        let mut function_value_map: BTreeMap<usize, Exp> = BTreeMap::new();
```

Comment:
Lift lambda expression and generate the corresponding spec function.

Reply:
Add as a comment.